### PR TITLE
Created ADOT configuration doc

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: integrations/kubecost-aws-distro-open-telemetry.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# # Kubecost with AWS Distro for OpenTelemetry (ADOT)
```
Comment (thomasvn) [inline integrations/kubecost-aws-distro-open-telemetry.md:1]:
```suggestion
# Kubecost with AWS Distro for OpenTelemetry (ADOT)
```

File: integrations/kubecost-aws-distro-open-telemetry.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### Kubecost Agent installation
+
+1. Create the Kubecost namespace:
+
+    ```bash
+    kubectl create ns YOUR_NAMESPACE
+    ```
+
+1. Create the AWS IAM policy to allow Kubecost to query metrics from AMP:
```
Comment (thomasvn) [inline integrations/kubecost-aws-distro-open-telemetry.md:37]: Is it worthwhile mentioning that AMP (Amazon Managed Prometheus) is a pre-requisite for this guide?

File: integrations/kubecost-aws-distro-open-telemetry.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+### ADOT configuration
+
+There are many options for deploying the ADOT daemonSet. At a minimum, Kubecost needs the provided scrape config to be added to the ADOT Prometheus ConfigMap. This sample ConfigMap also contains cAdvisor metrics, which is required by Kubecost.
+
+```bash
+kubectl apply -f example-configs/prometheus-daemonset.yaml
+```
+
+Alternatively, you can add these items to your [existing ConfigMap](https://github.com/kubecost/poc-common-configurations/blob/main/aws-amp/adot/example-configs/kubecost-adot-scrape-config.yaml).
```
Comment (thomasvn) [inline integrations/kubecost-aws-distro-open-telemetry.md:27]: A lot of the files in this doc (e.g. `prometheus-damonset.yaml`, `iam-read-amp-metrics.json`, `iam-kubecost-metrics-s3-policy.json`) are coming from the poc-common-configs repo. Should we make that more clear?

**Review by jessegoodier — APPROVED** [review]
See my last edit, feel free to reword and merge.

**Comment by jessegoodier** [comment]
Thanks for reviewing @thomasvn, 100% agreed.

**Pull request merged**